kinto/config/kinto.tpl

```diff
@@ -123,7 +123,7 @@ kinto.includes = kinto.plugins.default_bucket
 # kinto.logging_renderer = kinto.core.logs.ClassicLogRenderer
 
 [loggers]
-keys = root, kinto_core
+keys = root
```

Comment:
Here we lose the fact that we may want the DEBUG log level for kinto, while the INFO level for other package is fine.